src/api/app/views/webui2/webui/users/tasks/index.html.haml

```diff
@@ -0,0 +1,87 @@
+            Declined Requests (#{User.current.declined_requests.count})
+        %li.nav-item
+          %a#all-requests-tab{ href: '#all-requests', title: "All Requests from #{User.current.login}",
+            class: 'nav-link text-nowrap', 'data-toggle': 'tab', 'aria-controls': 'all-requests-tab', 'aria-selected': 'false', role: 'tab' }
+            All Requests
+        %li.nav-item.dropdown
+          %a.nav-link.dropdown-toggle{ href: '#', 'data-toggle': 'dropdown', 'role': 'button', 'aria-expanded': 'false', 'aria-haspopup': 'true' }
+          .dropdown-menu.dropdown-menu-right
+    .card-body
+      .tab-content
+        .tab-pane.fade.show.active#requests-in{ role: 'tabpanel', 'aria-labelledby': 'requests-in-tab' }
+          = render(partial: 'webui2/shared/requests_table', locals: { id: 'requests_in_table', source_url: user_requests_path(User.current) })
+        .tab-pane.fade#requests-out{ role: 'tabpanel', 'aria-labelledby': 'requests-out-tab' }
+          = render(partial: 'webui2/shared/requests_table', locals: { id: 'requests_out_table', source_url: user_requests_path(User.current) })
+        .tab-pane.fade#requests-declined{ role: 'tabpanel', 'aria-labelledby': 'requests-declined-tab' }
+          = render(partial: 'webui2/shared/requests_table', locals: { id: 'requests_declined_table', source_url: user_requests_path(User.current) })
+        .tab-pane.fade#all-requests{ role: 'tabpanel', 'aria-labelledby': 'all-requests-tab' }
+          = render(partial: 'webui2/shared/requests_table', locals: { id: 'all_requests_table', source_url: user_requests_path(User.current) })
+
+  - if User.current.involved_patchinfos.present?
+    .card.mt-3#patchinfos
+      .bg-light
+        %ul.nav.nav-tabs.pt-2.px-3.flex-nowrap.collapsible{ 'role': 'tablist' }
+          %li.nav-item
+            %a{ href: '#patchinfos-in', title: "Requests that #{User.current.login} has to merge", class: 'nav-link text-nowrap active' }
+              Maintenance Requests
+      .card-body
+        .tab-content#patchinfos-in
+          %table.responsive.table.table-striped.table-bordered#open-patchinfos-table{ title: 'Running Maintenance Updates', width: '100%' }
+            %thead
+              %tr
+                %th Project
+                %th Package
+                %th Issues
+                %th Actions
+            %tbody
+              - User.current.involved_patchinfos.each do |pi|
+                %tr
+                  - shortened_prj, shortened_pkg = elide_two(pi[:package][:project], pi[:package][:name], 60)
+                  %td= link_to(shortened_prj, project_show_path(pi[:package][:project]))
+                  %td= link_to(shortened_pkg, package_show_path(pi[:package][:project], pi[:package][:name]))
+                  %td
+                    - size = pi[:issues].size
+                    - pi[:issues].each_with_index do |item, index|
+                      = succeed "#{',' if index < size - 1}" do
```

Comment:
[nitpick] I would use `index <= size`

Reply:
:thinking:  I don't think so... If we have an array of 4 elements we want to go from 0 to 2, so `index <= size` would finish on 4 and `index < size` on 3...

Reply:
Don't we want to have all the issues? Because if we go from 0 to 2, we are missing one.

Reply:
This code adds a comma at the end of each element, instead of the last one.

Reply:
Oh sorry, then I said nothing :slightly_smiling_face: